### ISSUE TITLE
Add accessible label to drill down button

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#4394]
### What should we do?

The subprocess drill down button does not have any accessible label. Let's add it.

Report:

```json
{
  "id": "button-name",
  "impact": "critical",
  "tags": [
    "cat.name-role-value",
    "wcag2a",
    "wcag412",
    "section508",
    "section508.22.a",
    "TTv5",
    "TT6.a",
    "EN-301-549",
    "EN-9.4.1.2",
    "ACT"
  ],
  "description": "Ensures buttons have discernible text",
  "help": "Buttons must have discernible text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI",
  "nodes": [
    {
      "any": [
        {
          "id": "button-has-visible-text",
          "data": null,
          "relatedNodes": [],
          "impact": "critical",
          "message": "Element does not have inner text that is visible to screen readers"
        },
        {
          "id": "aria-label",
          "data": null,
          "relatedNodes": [],
          "impact": "critical",
          "message": "aria-label attribute does not exist or is empty"
        },
        {
          "id": "aria-labelledby",
          "data": null,
          "relatedNodes": [],
          "impact": "critical",
          "message": "aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty"
        },
        {
          "id": "non-empty-title",
          "data": {
            "messageKey": "noAttr"
          },
          "relatedNodes": [],
          "impact": "critical",
          "message": "Element has no title attribute"
        },
        {
          "id": "presentational-role",
          "data": null,
          "relatedNodes": [],
          "impact": "critical",
          "message": "Element's default semantics were not overridden with role=\"none\" or role=\"presentation\""
        }
      ],
      "all": [],
      "none": [],
      "impact": "critical",
      "html": "<button class=\"bjs-drilldown\">",
      "target": [
        ".bjs-drilldown"
      ],
      "failureSummary": "Fix any of the following:\n  Element does not have inner text that is visible to screen readers\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Element has no title attribute\n  Element's default semantics were not overridden with role=\"none\" or role=\"presentation\""
    }
  ]
}
```

### Why should we do it?

Child of https://github.com/bpmn-io/internal-docs/issues/909

**Comment by barmac** [comment]
This will be fixed via https://github.com/bpmn-io/bpmn-js/pull/2194
